extensions/adobe/experience/campaign/address.schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/address",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Address Extensions for Adobe Campaign",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/address"],
+  "description": "Address extension properties specific to Adobe Campaign.",
+  "definitions": {
+    "address": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/address/addressErrorCount": {
+          "title": "Address Error Count",
+          "type": "integer",
+          "description":
+            "Number of consecutive errors when sending to this address."
+        },
+        "https://ns.adobe.com/experience/campaign/address/addressQuality": {
+          "title": "Address Quality",
+          "type": "integer",
+          "description": "Address quality rating.",
+          "meta:enum": {
```

Comment:
If these are the only values you support, make the type a proper `enum` instead of an `integer`. Also: we typically use strings as enum keys instead of numbers, which is something we would do when moving this into the standard.